util/tls/tls_socket.cc

```diff
@@ -176,6 +192,7 @@ io::Result<size_t> TlsSocket::RecvMsg(const msghdr& msg, int flags) {
 
   Engine::MutableBuffer dest{reinterpret_cast<uint8_t*>(io->iov_base), io->iov_len};
   size_t read_total = 0;
+  SpinCounter spin_count(20);
```

Comment:
`20` is adhoc, looking for suggestions for the heuristic

Reply:
it's fine